front/js/product.js

```diff
@@ -0,0 +1,51 @@
+//The function below displays the product selected by the user on the home page
```

Comment:
is this code suposed to run when page load ? even if the answer is yes wrap everything in a function at list or even better try to split the different parts of your code in differents functions that are gonna be called by your main one which will be called at to of your file.


```

function <INSERT NAME HERE> (){
const whatProductUrl = window.location.href;
const productUrl = new URL(whatProductUrl);
const search_params = new URLSearchParams(productUrl.search); 

const data = <another function to call for triggering fetch >()
<another function to render product id based on previously received data>(data)
}

<INSERT NAME HERE>()
```